src/bin/vip-search-replace.js

```diff
@@ -12,33 +12,39 @@ const debug = debugLib( '@automattic/vip:bin:vip-search-replace' );
 const examples = [
 	// `search-replace` flag
 	{
-		usage: 'vip search-replace <file.sql> --search-replace="from,to"',
+		usage: 'vip search-replace file.sql --search-replace="from,to"',
 		description:
-			'Replace instances of <from> with <to> in the provided <file.sql>\n' +
-			'       * Ensure there are no spaces between your search-replace parameters',
+			'Search for every instance of the value "from" in the local input file named "file.sql" and replace it with the value "to".\n' +
+			'       * Results of the operation output to STDOUT by default.',
 	},
 	// `in-place` flag
 	{
-		usage: 'vip search-replace <file.sql> --search-replace="from,to" --in-place',
-		description: 'Perform Search and Replace explicitly on the provided input <file.sql> file',
+		usage: 'vip search-replace file.sql --search-replace="from,to" --in-place',
+		description:
+			'Perform the search and replace operation and save the results to the local input file "file.sql".',
 	},
 	// `output` flag
 	{
-		usage: 'vip search-replace <file.sql> --search-replace="from,to" --output="<output.sql>"',
+		usage: 'vip search-replace file.sql --search-replace="from,to" --output=output-file.sql',
 		description:
-			'Search and Replace to the specified output <output.sql> file\n' +
-			'       * Has no effect when the `in-place` flag is used',
+			'Perform the search and replace operation and save the results to a local clone of the input file named "output-file.sql".',
```

Comment:
It is unclear to me (from the previous description) where the output will go. Given the "Has no effect when the `in-place` flag is used," I would assume that the in-place operation would be performed on the source file (`file.sql`) and the result will be saved to that very file, not to `output-file.sql`.

Reply:
See https://github.com/Automattic/vip-cli/pull/2079/files#diff-1d697c6fa0eb15c3a87ec97840846b8674b75621357aac3c1e65681f03f25708R47

Reply:
One of the goals of the updated descriptions is to put any limitations or requirements for a command option in the description for the option, not in the usage example. 

I agree with you that the "Has no effect" phrase is unclear. What if we updated the description for `--output` to something like this:

```
.option(
		'output',
		'Save the results of the search and replace operation to a clone of the local input file. Accepts a local file path. Overridden by --in-place if both options are run in the same command.'
	)
```

Reply:
I'd suggest instead:

```
A local file name to save the results of the search and replace operation. Has no effect if the --in-place parameter is also included in the command.
```

Reply:
I think it's important to make it clear that the option will create the file; that the file does not need to pre-exist for the option to work. 🤔